exchangeAPI/src/handleErrors.js

```diff
@@ -0,0 +1,69 @@
+let symbolList = [];
```

Comment:
I think is not necessary to clarify that this is a List because you are already saying with this `= []` that it will be a list.